README.md

```diff
@@ -52,6 +52,23 @@ spec:
 
 This create a rule in which every time a user from the `corp-ldap` provider is created, a namespace called `<username>-sandbox` is also created.
 
+More advanced templating functions found in the popular k8s management tool [Helm](https://helm.sh/) is also available. This includes the [Sprig](http://masterminds.github.io/sprig/) templating library and the addition of the [lookup](https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-lookup-function) function used in Helm which allows for runtime access to cluster resources when processing templates.
```

Comment:
let's link the exact page in helm in which these functions are described. also elt's add the to/from yaml list that the helm docs never described.

Reply:
- I added the link to the templates documentation in helm that outlines all of the functions.
- Added a table that hows all the to/from functions
- When testing I ran the operator locally using this cli command `OPERATOR_NAME='namespace-configuration-operator' operator-sdk --verbose run  local --watch-namespace "" --operator-flags="--zap-level=debug"`
- I added an additional test file